### PR TITLE
Feature/keypoints support

### PR DESCRIPTION
**Pull request by BloodAxe**
- [x] Add support in Compose
- [x] Add support for Flip w/ tests
- [x] Add support for Scale w/ tests
- [x] Add support for Rotate w/ tests
- [x] Add support for Rotate90 w/ tests
- [x] Add support for ShiftScaleRotate w/ tests
- [x] Add support for RandomCrop w/ tests
- [x] Add support for CentralCrop w/ tests
- [ ] Update docs
- [x] Update showcase

### CODE REVIEW

File: tests/test_keypoint.py [hunk 1]
```diff
@@ -0,0 +1,243 @@
...
+
+
+def test_convert_keypoints_from_albumentations():
+    keypoints = [[0.2, 0.3, 0.6, 0.8], [0.3, 0.4, 0.7, 0.9, 99]]
+    image = np.ones((100, 100, 3))
+    converted_bboxes = convert_keypoints_from_albumentations(keypoints, rows=image.shape[0], cols=image.shape[1],
+                                                             target_format='xyas')
+    converted_bbox_1 = convert_keypoint_from_albumentations(keypoints[0], rows=image.shape[0], cols=image.shape[1],
+                                                            target_format='xyas')
+    converted_bbox_2 = convert_keypoint_from_albumentations(keypoints[1], rows=image.shape[0], cols=image.shape[1],
```
Comment (albu) [inline tests/test_keypoint.py]:
```suggestion
    converted_keypoint_2 = convert_keypoint_from_albumentations(keypoints[1], rows=image.shape[0], cols=image.shape[1],
```

File: tests/test_keypoint.py [hunk 2]
```diff
@@ -0,0 +1,243 @@
...
+                                                            target_format='xyas')
+    assert converted_bboxes == [converted_bbox_1, converted_bbox_2]
```
Comment (albu) [inline tests/test_keypoint.py]:
```suggestion
    assert converted_keypoints == [converted_keypoint_1, converted_keypoint_2]
```

**Comment by adizhol** [comment]
@BloodAxe 
I think we need to add support for normalized xy coordinates.

**Comment by adizhol** [comment]
@BloodAxe
And I think there is an issue with Rotate, because it probably crop the augmented image to the original size, causing keypoints to fall outside the image (negative xy values or xy values larger than image size).
I guess it can be fixed if you resize the rotated image instead of cropping?

**Comment by adizhol** [comment]
@BloodAxe  @albu 
How to apply ImageOnlyTranform?
I'm getting NotImplemented exceptions

thanks

**Pull request merged**